README.md

```diff
@@ -57,7 +57,8 @@ You can look at different show cases for it here as [Component](https://rawgit.c
            myListFormatter(data: any): string {
               return `(${data[key]}) ${data[name]}`;
             }
-
+  * **`outer-input-element`**, HTMLInputElement reference, path any input from outer scope.
+  * **`extFormControl`**, AbstructControl, path your own FormControl object, can be used with outer-input-element.
```

Comment:
@ctrahhuk did you mean "AbstractControl" and again "pass your own FormControl object"